### ISSUE TITLE
Cann't install chromVAR successfully on Linux

### ISSUE THREAD

**Issue by Wenhui-bio** [GreenleafLab/chromVAR#91]
Hi, I have tried some times but not install this successfully yet.  Can anyone help me?

first, I create a conda environment: `conda create -n r`
and  then activate r to install R and packages: `conda install -c conda-forge r-base` #R4.1
finally, try to install chromVAR with this command, `BiocManager::install("chromVAR")`
but I got the error information as follows
```
1: In .inet_warning(msg) :
  installation of package ‘RSQLite’ had non-zero exit status
2: In .inet_warning(msg) :
  installation of package ‘AnnotationDbi’ had non-zero exit status
3: In .inet_warning(msg) :
  installation of package ‘annotate’ had non-zero exit status
4: In .inet_warning(msg) :
  installation of package ‘GO.db’ had non-zero exit status
5: In .inet_warning(msg) :
  installation of package ‘CNEr’ had non-zero exit status
6: In .inet_warning(msg) :
  installation of package ‘TFBSTools’ had non-zero exit status
7: In .inet_warning(msg) :
  installation of package ‘chromVAR’ had non-zero exit status
```

before using R created by conda, I have tried installation chromVAR so many times by R installed by source code in my linux account, but unfortunately I also got some errors, such as `fatal error: gsl/gsl_rng.h: No such file or directory`, I guess may be I have lacked some shared file in my R program, so I newly create a new conda env and install R, but I still cann't install this(the error information are above)

**Comment by Wenhui-bio** [comment]
Hi, this installation problem has solved.
cann't install RSQLite by `install.packages("RSQLite")`, try to used a low RSQLite version.
```
require(devtools)
install_version("RSQLite", version = "2.2.5")
```
and then `BiocManager::install("chromVAR")`

**Comment by candacelei** [comment]
Hi, 
Did you successfully install chromVAR? I got the following error message when I followed your step:
> install_version("RSQLite", version = "2.2.5")
--- Please select a CRAN mirror for use in this session ---
Error in download_version_url(package, version, repos, type) :
  version '2.2.5' is invalid for package 'RSQLite'

**Comment by Wenhui-bio** [comment]
Yes, I installed it successfully. May be you need to debug according to all errors emerged.

**Comment by clairecheng0906** [comment]
You can install RSQLie 2.25 by remotes. 

```
library(remotes)
remotes::install_version("RSQLite", version = "2.2.5")
```
